apps/meteor/client/hooks/useAppActionButtons.ts

```diff
@@ -36,6 +36,26 @@ export const useAppActionButtons = <TContext extends `${UIActionButtonContext}`>
 		[],
 	);
 
+	useEffect(() => {
+		if (!uid) {
+			return;
+		}
+
+		// Setup Tracker to listen to Meteor's status changes
+		const statusTracker = Tracker.autorun(() => {
+			const isOffline = !Meteor.status().connected;
```

Comment:
Hey @tassoevan we were talking about trying to have some kind of less-indirect way of telling when the socket disconnected, I found that Meteor.status is what we used in similar situations so I decided to try to implement using it here, but if you thing this approach is not good I guess we can try to find something different